tests/unit/models/transactions/test_did_set.py

```diff
@@ -62,3 +62,46 @@ def test_empty(self):
                 error.exception.args[0],
                 "{'did_set': 'Must have one of `did_document`, `data`, and `uri`.'}",
             )
+
+    def test_all_empty_fields(self):
+        with self.assertRaises(XRPLModelException):
+            DIDSet(account=_ACCOUNT, data="", did_document="", uri="")
+
+    def test_delete_multiple_fields(self):
```

Comment:
super nit:
```suggestion
    def test_remove_multiple_fields(self):
```
I was a bit confused at first because I thought this might be a `DIDDelete` test.